shotover-proxy/src/transforms/cassandra/sink_cluster/node_pool.rs

```diff
@@ -73,6 +73,21 @@ impl NodePool {
         write_lock.insert(id, metadata);
     }
 
+    pub fn get_shuffled_nodes_in_dc_rack(
+        &mut self,
```

Comment:
Do we need to collect twice? This can just return a list of the shuffled addresses.

Reply:
Good idea